akka-actor/src/main/scala/akka/routing/ScatterGatherFirstCompleted.scala

```diff
@@ -54,9 +54,9 @@ private[akka] final case class ScatterGatherFirstCompletedRoutees(
       val promise = Promise[Any]()
       routees.foreach {
         case ActorRefRoutee(ref) =>
-          promise.tryCompleteWith(ref.ask(message))
+          promise.completeWith(ref.ask(message))
         case ActorSelectionRoutee(sel) =>
-          promise.tryCompleteWith(sel.ask(message))
+          promise.completeWith(sel.ask(message))
```

Comment:
is this correct? I think it sends out many requests and whatever is completed first will complete the promise, but other replies may arrive later